formatter_filters/cloudevents/formatter_filter.go

```diff
@@ -86,8 +86,10 @@ type FormatterFilter struct {
 	Format Format
 
 	// Predicate is a func that returns true if we want to keep the cloudevent.
-	// The interface{} parameter will be a CloudEvent struct.
-	Predicate func(cloudevent interface{}) (bool, error)
+	// The context parameter is the context of Process(ctx context.Context, e
+	// *eventlogger.Event) and the interface{} parameter will be a CloudEvent
+	// struct.
+	Predicate func(ctx context.Context, cloudevent interface{}) (bool, error)
```

Comment:
`cloudevent` is probably more descriptive, but optionally it can be changed to `event` since the struct was renamed.  Could also just make it `e`, but this is very minor feel free to ignore...

Reply:
ty for pointing this out... changing it to `cloudevents.Event struct` which makes it very explicit